base/cmd.jl

```diff
@@ -294,6 +304,38 @@ function addenv(cmd::Cmd, env::Vector{<:AbstractString}; inherit::Bool = true)
     return addenv(cmd, Dict(k => v for (k, v) in eachsplit.(env, "=")); inherit)
 end
 
+"""
+    setcpuaffinity(original_command::Cmd, cpus) -> command::Cmd
+
+Set the CPU affinity of the `command` by a list of CPU IDs (1-based) `cpus`.  Passing
+`cpus = nothing` means to unset the CPU affinity if the `original_command` has any.
+
+This function is tested only on Linux.
```

Comment:
`tested` or `supported`?

Reply:
```suggestion
This function is supported only in Linux and Windows.  It is not supported in macOS because
libuv does not support affinity setting.
```